src/site/constants/drupals.js

```diff
@@ -29,4 +30,17 @@ const DRUPALS = {
   [ENVIRONMENTS.VAGOVPROD]: DRUPAL_LIVE,
 };
 
+const ENABLED_ENVIRONMENTS = new Set([
+  ENVIRONMENTS.LOCALHOST,
+  ENVIRONMENTS.VAGOVDEV,
+  ENVIRONMENTS.VAGOVSTAGING,
+]);
+
+const PREFIXED_ENVIRONMENTS = new Set([
+  // ENVIRONMENTS.LOCALHOST,
+  ENVIRONMENTS.VAGOVSTAGING,
+]);
```

Comment:
I'm not sure if localhost should apply the prefix or not, but I decided it was best to mirror dev.